README.md

```diff
@@ -125,11 +126,12 @@ $ go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
 ```
 
 Unfortunately, it has become necessary to lock dependencies versions to obtain a
-clean build of wsk.  To that end, it's now necessary to populate the `vendors`
```

Comment:
Can you remove unfortunately it has become... seems unnecessary and is a best practice to pin

Reply:
Sure

Reply:
Actually, this pull request does that perfectly, maybe we'd like to merge that in to reduce merge conflicts if I also fix it: https://github.com/apache/incubator-openwhisk-cli/pull/361